### DISCUSSION TITLE
ScrollView vs ScrollViewer

### DISCUSSION THREAD

**Discussion by JoeTomkinson** [microsoft/microsoft-ui-xaml#9144] [Q&A] [answered]
Legitimately, can someone in the know tell me the actual difference in use cases for both of these controls in WinUI3. Either I've missed some documentation somewhere that clearly defines it, or the Microsoft Docs just kind of say here's both enjoy.

The reason for asking is that I'm using an Inverted Listview and the Community Toolkit exstension method for the ListView looks like under the hood it assumes a child control of ScrollViewer when using it's methods for essentially "Scroll To". However as ScrollView is only available for WinUI3 according to the microsoft docs my brain is telling me it's the better option, that coupled with some weird scroll issues I've seen with the scrollviewer I want to make the move across but not if it's the wrong overall choice.

**Comment by kmgallahan** [comment, marked as answer]
https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.scrollview

> The ScrollView control is similar in behavior and usage to the [ScrollViewer](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.scrollviewer?view=windows-app-sdk-1.4) control, but is based on [InteractionTracker](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.composition.interactions.interactiontracker?view=windows-app-sdk-1.4), has new features such as animation-driven view changes, and is designed to ensure full functionality with [ItemsRepeater](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.itemsrepeater?view=windows-app-sdk-1.4).

**Reply by JoeTomkinson** [reply]
Legend, thank you.
